### PR TITLE
KPIV-17: Add wait-mod to conference flags

### PR DESCRIPTION
**Pull request by dhuynh-Kazoo**
This flag forces all participants to wait until the moderator joins so the conference can be started. Therefore, participants should dial into the conference with wait-mod in the conference flags, e.g., <<"+flags{wait-mod}">>

### CODE REVIEW

**Comment by dhuynh-Kazoo** [comment]
should be linked to ecallmgr with these commits: 
https://github.com/2600hz/kazoo-ecallmgr/pull/255/commits/693367e6bf910cf6c729044fffc7a8c2fc68229d
https://github.com/2600hz/kazoo-core/pull/840/commits/b80919f69ebb2732aa8165a3f136b8e59a707332

File: src/conf_participant.erl [hunk 1]
```diff
@@ -623,6 +623,7 @@ send_conference_command(Conference, Call) ->
                   [{<<"Mute">>, is_muted(Conference)}
                   ,{<<"Deaf">>, is_deaf(Conference)}
                   ,{<<"Moderator">>, kapps_conference:moderator(Conference)}
+                  ,{<<"Wait-Moderator">>, kapps_conference:wait_for_moderator(Conference)}
```
Comment (icehess) [inline src/conf_participant.erl]: Why this change? wait for moderator is not a member flag. It is a conference flags and is set when the conference session is created. look at `conf_config_req:verify_wait_mode/2`.

**Review by icehess — COMMENTED** [review]
Please don't mix unrelated changes in a  single change. (fixing wait mod and adding conference recording stuff)

File: src/conf_participant.erl (hunk 1 above)
Comment (dhuynh-Kazoo) [reply src/conf_participant.erl]: Hi @icehess, my understanding from the `conf_participant:send_conference_command/2` is that we are trying to use the [Conference Dial Application](https://developer.signalwire.com/freeswitch/FreeSWITCH-Explained/Modules/mod_conference_3965534/#conference-dialplan-application) when a participant connects to the conference. So it’s okay to add the `wait-mod or moderator` flag to the dial command. It seems my code might be confusing because we had member-flags in the profile.

So let me remove them and add `wait-mod` and `moderator `to the member-flags in the conference profile.
